### PR TITLE
network/wi-fi: move vwifi initialization

### PR DESCRIPTION
**Pull request by pillip8282**
vwifi was initialized a user space. So it caused build fail if
loadable configuration was set. Therefore change the location of
initialization of vwifi device.

### CODE REVIEW

File: os/drivers/wireless/virtual/vdev.c [hunk 1]
```diff
@@ -213,13 +216,109 @@ int _vwifi_create_msgqueue(int *fd)
...
+		}
+
+		res = vwifi_handle_message(msg.req);
+		if (res < 0) {
+			VWIFI_ERROR(res);
+			break;
+		}
+		_send_signal(&msg);
+	}
+	return;
```
Comment (jeongchanKim) [inline os/drivers/wireless/virtual/vdev.c:295]: I think this return is not needed.

Comment (pillip8282) [reply os/drivers/wireless/virtual/vdev.c:295]: I think this is trivial. so I won't remove it

File: os/drivers/wireless/virtual/vdev.c [hunk 2]
```diff
@@ -213,13 +216,109 @@ int _vwifi_create_msgqueue(int *fd)
 	return 0;
 }
 
+/*
+ * Internal Function
+ */
+static struct netdev* vdev_register_dev(int sizeof_priv)
```
Comment (davidfather) [inline os/drivers/wireless/virtual/vdev.c:222]: When is the input param 'sizeof_priv' used? If it is useless, please remove it.

**Review by jeongchanKim — DISMISSED** [review]
LGTM

**Comment by jeongchanKim** [comment]
Please check the coding rule.

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by an4967 — APPROVED** [review]
LGTM

**Pull request merged**